packages/vite/src/node/__tests__/build.spec.ts

```diff
@@ -4,8 +4,8 @@ import colors from 'picocolors'
 import { describe, expect, test, vi } from 'vitest'
 import type { OutputChunk, OutputOptions, RollupOutput } from 'rollup'
 import type { LibraryFormats, LibraryOptions } from '../build'
+import { build } from '..'
 import {
-  build,
```

Comment:
What circular import exist here? It is because we override `vitest > vite` to the local package.

Reply:
In this case, Vitest tries to evaluate from `packages/vite/src/node/build.ts` and that seems to be broken too as seen by tsx.

```
$ pnpm exec tsx packages/vite/src/node/build.ts
/home/hiroshi/code/others/vite/packages/vite/src/node/config.ts:616
  build: buildEnvironmentOptionsDefaults,
         ^
ReferenceError: Cannot access 'buildEnvironmentOptionsDefaults' before initialization
...
```

I think common cases are `src/node/build.ts` or `src/node/server/index.ts`, so I replaced those import with `src/node/index.ts`.